src/client/js/partials/settings.js

```diff
@@ -12,7 +12,10 @@ function handleEvent (e) {
   switch (true) {
     case e.type === 'email-added':
       document.querySelector('dialog[data-partial="add-email"]')
-        .addEventListener('close', () => window.location.reload(), { once: true })
+        .addEventListener('close', () => {
+          window.gtag('event', 'Add Email', { action: 'click', page_location: location.href })
+          window.location.reload()
+        }, { once: true })
```

Comment:
This is actually listening for the close event of the dialog box, so I think this would fire off an incorrect event.

If you want to track that a user has clicked the "add email" button, that might be elsewhere – but I'd imagine button clicks are already tracked by default?  If so, we could probably just delete this gtag addition.

Reply:
Hm, I think only outbound clicks are tracked by default, we have "enhanced event measurment" enabled: https://support.google.com/analytics/answer/9216061

But yes that's a typo... let me double-check this.

Reply:
Hm, yeah I actually already moved this to the `add-email` module, so I think this can just be removed.